.github/workflows/test-wdio.yml

```diff
@@ -1,21 +1,14 @@
 name: WebdriverIO Tests
 
 on:
-  merge_group:
-  pull_request:
-  push:
-    branches:
-      - 'main'
+  workflow_call:
```

Comment:
Is this action getting called from anywhere now?

Reply:
good point, it is missing in the main test workflow

Reply:
added in https://github.com/ionic-team/stencil/pull/5930/commits/663118dddbe7425dba404eb8fee4ca382c371417